### ISSUE TITLE
Segmentation fault with prior scipy import

### ISSUE THREAD

**Issue by fellhorn** [intel-isl/Open3D#1426]
**Describe the bug**
Open3D causes a segmentation fault if there is a prior import from scipy.spatial.
If scipy.spatial is imported afterwards there is no segfault and open3d works as intended.

**To Reproduce**
```
Python 3.7.6 (default, Jan  3 2020, 23:24:26) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from scipy.spatial import cKDTree
>>> import open3d
Segmentation fault (core dumped)
```

BUT

```
Python 3.7.6 (default, Jan  3 2020, 23:24:26) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import open3d
>>> from scipy.spatial import cKDTree
>>>
```

**Environment (please complete the following information):**
 - OS: Debian 10.2 (Buster)
 - Python version: 3.7.6
 - Open3D version: 0.9.0
 - Is this remote workstation?: yes (Docker in Kubernetes)
 - How did you install Open3D?: pip

**Comment by griegler** [comment]
Might be related to #1217 #1207 #922.

**Comment by syncle** [comment]
FYI, We have [a PR on this ](https://github.com/intel-isl/Open3D/pull/1262).

**Comment by theNded** [comment]
Fixed as of 0.12.0. Closed for now.
